### PR TITLE
Fix paged block store tier name

### LINKED ISSUE #15930: NPE at DefaultStorageTierAssoc.getOrdinal when using paged block store
**Alluxio Version:**
What version of Alluxio are you using?
current master

**Describe the bug**

When using the paged block store, block master cannot find a tier in the global tier list. This causes a NPE.

```
2022-07-21 09:25:28,311 ERROR FileSystemMasterClientServiceHandler - Exit (Error): ListStatus: request=path: "/"
options {
  loadMetadataType: ONCE
  commonOptions {
    syncIntervalMs: -1
    ttl: -1
    ttlAction: DELETE
  }
  recursive: true
  loadMetadataOnly: false
}

java.lang.NullPointerException
 at alluxio.DefaultStorageTierAssoc.getOrdinal(DefaultStorageTierAssoc.java:99)
 at alluxio.master.file.DefaultFileSystemMaster.isInTopStorageTier(DefaultFileSystemMaster.java:2401)
 at alluxio.master.file.DefaultFileSystemMaster.getFileInMemoryPercentageInternal(DefaultFileSystemMaster.java:2351)
 at alluxio.master.file.DefaultFileSystemMaster.getFileInfoInternal(DefaultFileSystemMaster.java:965)
 at alluxio.master.file.DefaultFileSystemMaster.listStatusInternal(DefaultFileSystemMaster.java:1213)
 at alluxio.master.file.DefaultFileSystemMaster.listStatusInternal(DefaultFileSystemMaster.java:1201)
 at alluxio.master.file.DefaultFileSystemMaster.listStatusInternal(DefaultFileSystemMaster.java:1201)
 at alluxio.master.file.DefaultFileSystemMaster.listStatus(DefaultFileSystemMaster.java:1124)
 at alluxio.master.file.FileSystemMasterClientServiceHandler.lambda$listStatus$10(FileSystemMasterClientServiceHandler.java:255)
 at alluxio.RpcUtils.callAndReturn(RpcUtils.java:123)
 at alluxio.master.file.FileSystemMasterClientServiceHandler.listStatus(FileSystemMasterClientServiceHandler.java:253)

<snip>
```

**To Reproduce**
Use paged block store and do a `ls` on a file.

**Additional context**

Current tier of paged block store is `DefaultTier`, but `alluxio.master.block.DefaultBlockMaster#MASTER_STORAGE_TIER_ASSOC` expects it to be one of `MEM` `SSD` `HDD`.

### PR DESCRIPTION
**Pull request by dbw9580**
Fixes #15930

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStoreMeta.java [hunk 1]
```diff
@@ -41,9 +44,11 @@
...
  */
 public class PagedBlockStoreMeta implements BlockStoreMeta {
-  public static final String DEFAULT_TIER = "DefaultTier";
-  public static final String DEFAULT_DIR = "DefaultDir";
-  public static final String DEFAULT_MEDIUM = "SSD";
+  // Paged block store does not support multiple tiers, so assume everything is in the top tier
+  public static final String DEFAULT_TIER =
+      Configuration.getString(PropertyKey.MASTER_TIERED_STORE_GLOBAL_LEVEL0_ALIAS);
+  // top tier is conventionally ramdisk
+  public static final String DEFAULT_MEDIUM = Constants.MEDIUM_MEM;
```
Comment (dbw9580) [inline core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStoreMeta.java:51]: Although I suspect in practice the page cache will not be in ramdisk, but for a paged block store in worker that needs to play nice with master, this convention still needs to be upheld, e.g. `alluxio.master.file.DefaultFileSystemMaster#getFileInMemoryPercentageInternal` checks  the in-memory percentage of a file by checking its blocks stored in the top tier.

**Comment by dbw9580** [comment]
@beinan can you please take a look?

**Review by beinan — APPROVED** [review]
lgtm

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**